module4/frontend/dist/bundle.js

```diff
@@ -130,7 +130,7 @@ var Module1 = /*#__PURE__*/function (_React$Component) {
   _createClass(Module1, [{
     key: "render",
     value: function render() {
-      console.log('rendering Module1');
+      // console.log('Rendering Module1');
```

Comment:
probably good idea to delete all those commented out console.logs for readability